etc/ATSCam/primary_hdu.header

```diff
@@ -3,56 +3,56 @@ BITPIX  =                    8 / bits per data value
 NAXIS   =                    0 / number of axes
 EXTEND  =                    T / Extensions are permitted
 ORIGIN  = 'LSST DM Header Service'/ FITS file originator
-DATE    = ''                   / Creation Date and Time of File
-DATE-OBS= ''                   / Date of the observation (image acquisition)
-MJD     =                -99.0 / Modified Julian Date that the file was written
-MJD-OBS =                -99.0 / Modified Julian Date of observation
+DATE    =                      / Creation Date and Time of File
+DATE-OBS=                      / Date of the observation (image acquisition)
+MJD     =                      / Modified Julian Date that the file was written
+MJD-OBS =                      / Modified Julian Date of observation
 OBS-LONG=           -70.749417 / [deg] Observatory east longitude
 OBS-LAT =           -30.244639 / [deg] Observatory latitude
 OBS-ELEV=               2663.0 / [m] Observatory elevation
```

Comment:
General comment, but we have to add in OBSGEO headers reasonably soon.

Reply:
Will add this to audit ticket